auth-web/src/components/auth/common/NumberedCompanyTooltip.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <v-tooltip top max-width="450px" light content-class="tooltip">
+    <template v-slot:activator="{ on }">
+      <span v-on="on"><a><u>numbered Benefit Company</u></a></span>
+    </template>
+    <v-card class="tooltip-content">
+      <h3 class="mb-3">Numbered Benefit Company</h3>
+      <span>Lorem ipsum... some definition of a numbered benefit company would go into this slot here</span>
+    </v-card>
+  </v-tooltip>
+</template>
+
+<script lang="ts">
+import { Component } from 'vue-property-decorator'
+import Vue from 'vue'
+
+@Component({})
```

Comment:
good job in separating this as a individual component  .. :)

Reply:
👍 Thanks!